examples/experimental/redis.js

```diff
@@ -1,87 +1,83 @@
-import { check } from "k6";
-import http from "k6/http";
-import redis from "k6/experimental/redis";
-import exec from "k6/execution";
-import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.1/index.js";
+import { check } from 'k6';
+import http from 'k6/http';
+import redis from 'k6/experimental/redis';
+import exec from 'k6/execution';
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
 export const options = {
 	scenarios: {
 		redisPerformance: {
-			executor: "shared-iterations",
+			executor: 'shared-iterations',
 			vus: 10,
 			iterations: 200,
-			exec: "measureRedisPerformance",
+			exec: 'measureRedisPerformance',
 		},
 		usingRedisData: {
-			executor: "shared-iterations",
+			executor: 'shared-iterations',
 			vus: 10,
 			iterations: 200,
-			exec: "measureUsingRedisData",
+			exec: 'measureUsingRedisData',
 		},
 	},
 };
-// Get the redis instance(s) address and password from the environment
-const redis_addrs = __ENV.REDIS_ADDRS || "";
-const redis_password = __ENV.REDIS_PASSWORD || "";
```

Comment:
Do we no longer need these?

Reply:
I can remove them if needed, I mostly adapted the old example to use the new service, whether the example itself was correct already I am not sure.